ts/DataGrid/DataGridOptions.d.ts

```diff
@@ -145,6 +153,20 @@ export interface ColumnOptions {
     show?: boolean;
 }
 
+/**
+ * Returns a formatted call's string.
+ */
+export interface CellFormatterCallback {
+    (this: CellValue): string;
```

Comment:
The function scope makes it impossible to use arrow functions as callbacks. I would expect that the table value comes as the first or second argument.

Reply:
We do not support arrow functions in Highcharts or Dashboards events and formatted.

Eg. https://jsfiddle.net/2g7edxvf/1/

Knowing that the formatter is a temporary solution (like we discussed during the last meeting), I would skip arrow function support.